ntcore/src/main/native/cpp/LocalStorage.cpp

```diff
@@ -130,7 +130,9 @@ void LocalStorage::Impl::NotifyTopic(TopicData* topic,
     if (!m_dataloggers.empty()) {
       auto now = Now();
       for (auto&& datalogger : m_dataloggers) {
-        if (wpi::starts_with(topic->name, datalogger->prefix)) {
+        if (wpi::starts_with(topic->name, datalogger->prefix) &&
+            (!datalogger->prefix.empty() ||
+             !wpi::starts_with(topic->name, '$'))) {
```

Comment:
`'$'` should already be stored as a constant somewhere, use the constant.

Reply:
Hmm?  '$' is a constant.  Or are you looking for a `constexpr char kMetaPrefix = '$'`?  There isn't one currently.